fission/src/ui/panels/DebugPanel.tsx

```diff
@@ -70,6 +71,30 @@ const DebugPanel: React.FC<PanelImplProps<void, void>> = ({ panel }) => {
                 <Button onClick={() => PreferencesSystem.clearPreferences()} className="w-full">
                     Clear Preferences
                 </Button>
+                <Button
+                    onClick={() => {
+                        openModal(
+                            ConfirmModal,
+                            {
+                                title: "Clear All Data",
+                                message:
+                                    "Are you sure you want to clear all preferences and cached data? This cannot be undone.",
+                                acceptText: "Clear & Reload",
+                                cancelText: "Cancel",
+                                onConfirm: () => {
+                                    window.localStorage.clear()
+                                    sessionStorage.clear()
+                                    window.location.reload()
+                                    console.log("All data cleared")
+                                },
```

Comment:
Should also clear the OPFS caches for mirabuf stuff

```suggestion
                                onConfirm: async () => {
                                    window.localStorage.clear()
                                    sessionStorage.clear()
                                    await navigator.storage.getDirectory().then(async (root) => {
                                        for await (const key of root.keys()) {
                                            await root.removeEntry(key, {recursive:true})
                                        }
                                    }).catch(() => {console.warn("couldn't empty opfs")})
                                    window.location.reload()
                                    console.log("All data cleared")
                                },
```